### PR TITLE
Cli command for retrieve

### PR DESCRIPTION
**Pull request by asutula**
Simple, writes a file

### CODE REVIEW

File: exe/fil-tools/cmd/deals_retrieve.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	"path"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	retrieveCmd.Flags().StringP("address", "a", "", "wallet address of the file to fetch")
```
Comment (jsign) [inline exe/fil-tools/cmd/deals_retrieve.go]: Nit: maybe confusing descripton? Alternative: `wallet address to use funds for retrieval`.

File: exe/fil-tools/cmd/deals_retrieve.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	retrieveCmd.Flags().StringP("cid", "c", "", "cid of the file to fetch")
```
Comment (jsign) [inline exe/fil-tools/cmd/deals_retrieve.go]: Nit: maybe te concept of _file_ can be avoided, in general. Not sure though.

File: exe/fil-tools/cmd/deals_retrieve.go (hunk 1 above)
Comment (asutula) [reply exe/fil-tools/cmd/deals_retrieve.go]: cool, updated.

File: exe/fil-tools/cmd/deals_retrieve.go (hunk 2 above)
Comment (asutula) [reply exe/fil-tools/cmd/deals_retrieve.go]: Sure. Changed to "data".

**Pull request merged**